src/components/ResourceForm/AccessLevelSelector.tsx

```diff
@@ -68,11 +67,11 @@ const AccessLevelSelector = ({
       >
         {accessLevels.map((accessLevel) => (
           <FormControlLabel
-            value={accessLevel.val}
+            value={accessLevel.roleId.toString()}
```

Comment:
nit: can't the `value` prop simply take a number? is this necessary?

Reply:
hm I vaguely recall I put this in because it wasn't working with `number`, then I checked the docs and it said `string` was required. but now I just checked again and it says `any` 🤔

will need to verify this again

Reply:
Just looked into it, this is actually necessary because `RadioGroup`'s value prop casts it to a string by default. Thus the current value doesn't show up as selected unless you also cast this to a string.